### PR TITLE
Allow to get handlers programmatically

### PR DESCRIPTION
**Pull request by piranna**
Console output is put on the cli, making the library to return the raw
handler objects since libraries should not use the console at all.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,27 +2,30 @@
 
 Node is running but you don't know why? why-is-node-running is here to help you.
 
-```
-$ npm install why-is-node-running
+```sh
+npm install why-is-node-running
```
Comment (yoshuawuyts) [inline README.md]: I'd leave stylistic disagreements as is; prepending shell commands with `$` is quite common

File: README.md [hunk 2]
```diff
@@ -2,27 +2,30 @@
...
   server.listen(0)
 }
 
+
 createServer()
 createServer()
 
 setTimeout(function () {
-  log() // logs out active handles that are keeping node running
+  console.error(why())  // logs out active handles that are keeping node running
```
Comment (yoshuawuyts) [inline README.md]: probably `s/why/log`; there's no reference to `why` in this example

**Comment by yoshuawuyts** [comment]
Thanks for your PR! 

I don't agree with merging `cli.js` + `index.js` -> `server.js`; I feel there are a lot of changes in this PR that don't contribute to the core premise of making handler programmatically available. Could revert the moving of files and stylistic changes? Thanks!

**Comment by mafintosh** [comment]
Closing cause very old

**Pull request closed without merging**